src/Helios.sol

```diff
@@ -106,26 +129,28 @@ contract Helios is OwnedThreeStep(tx.origin), SafeMulticallable, ERC1155, ERC115
 
         require(address(swapper).code.length != 0, "Helios: INVALID_SWAPPER");
 
+        uint112 token0amount;
```

Comment:
Changes here are due to forge coverage complaining for stask too deep.

I believe forge coverage turns off via-ir which moves code around and that would make the coverage report noisy.

In any case, I can move these to a separate PR if necessary.